lib/opentok/session.rb

```diff
@@ -28,6 +28,7 @@ module OpenTok
   #   Generates a token.
   #
   #   @param [Hash] options
+  #   @option options [String] :token_type The type of token to generate. Must be one of 'T1' or 'JWT'.
```

Comment:
@jeffswartz I think we'll need to update the OT docs [here](https://tokbox.com/developer/sdks/ruby/) and [here](https://tokbox.com/developer/guides/create-token/ruby/) to cover this new param.